### PR TITLE
Foldable2v refactoring

### PR DESCRIPTION
**Pull request by gcanti**

### CODE REVIEW

File: src/Foldable2v.ts [hunk 1]
```diff
@@ -116,17 +116,17 @@ export function getFoldableComposition<F, G>(F: Foldable2v<F>, G: Foldable2v<G>)
...
 }
 
-export function fold<F extends URIS3, M>(F: Foldable2v3<F>, M: Monoid<M>): <U, L>(fa: Type3<F, U, L, M>) => M
-export function fold<F extends URIS3, M, U, L>(F: Foldable2v3C<F, U, L>, M: Monoid<M>): (fa: Type3<F, U, L, M>) => M
-export function fold<F extends URIS2, M>(F: Foldable2v2<F>, M: Monoid<M>): <L>(fa: Type2<F, L, M>) => M
-export function fold<F extends URIS2, M, L>(F: Foldable2v2C<F, L>, M: Monoid<M>): (fa: Type2<F, L, M>) => M
-export function fold<F extends URIS, M>(F: Foldable2v1<F>, M: Monoid<M>): (fa: Type<F, M>) => M
-export function fold<F, M>(F: Foldable2v<F>, M: Monoid<M>): (fa: HKT<F, M>) => M
+export function fold<M, F extends URIS3>(M: Monoid<M>, F: Foldable2v3<F>): <U, L>(fa: Type3<F, U, L, M>) => M
+export function fold<M, F extends URIS3, U, L>(M: Monoid<M>, F: Foldable2v3C<F, U, L>): (fa: Type3<F, U, L, M>) => M
```
Comment (raveclassic) [inline src/Foldable2v.ts]: The order is changed for `*C` overloadings, this is a breaking change

Comment (raveclassic) [inline src/Foldable2v.ts]: The order of type arguments is changed for `*C` overloadings, this is a breaking change

Comment (raveclassic) [inline src/Foldable2v.ts]: The order of type arguments is changed for `*C` overloadings, this is a breaking change

Comment (gcanti) [reply src/Foldable2v.ts]: @raveclassic not sure what you mean, this module is **new**

Comment (raveclassic) [reply src/Foldable2v.ts]: Yeah, github was lagging and I though I deleted this message

**Pull request merged**